@navikt/core/react/src/util/i18n/locales/nn.ts

```diff
@@ -37,4 +37,10 @@ export default {
       labelSuffix: "slett",
     },
   },
+  Search: {
+    clear: "Tøm",
+    Button: {
+      search: "Søk",
+    },
+  },
```

Comment:
We could discuss if it's necessary to have `search` in a nested object 🤔

Reply:
Would this be a little easier to use?
```suggestion
  Search: {
    clearButton: "Tøm",
    searchButton: "Søk",
  },
```

Reply:
```suggestion
  Search: {
    clear: "Tøm",
    search: "Søk",
  },
```